programme/stellwerk.lua

```diff
@@ -102,6 +107,9 @@ local function zeichneSignal(signal, art)
         if art == SIGNAL_ART_HP then
             bildschirm.zeichneElement(signal, farbeUnten, "|")
             bildschirm.zeichneElement(signal, farbeOben, ">", 1)
+        elseif art == SIGNAL_ART_HPA then
+            bildschirm.zeichneElement(signal, farbeUnten, "|")
+            bildschirm.zeichneElement(signal, farbeOben, ">", 1)
```

Comment:
```suggestion
        if art == SIGNAL_ART_HP or art == SIGNAL_ART_HPA then
            bildschirm.zeichneElement(signal, farbeUnten, "|")
            bildschirm.zeichneElement(signal, farbeOben, ">", 1)
```
Zusammenfassen für weniger duplizierten Code.